src/app/beer_garden/command_publishing_blocklist.py

```diff
@@ -0,0 +1,50 @@
+from brewtils.models import Operation
+
+import beer_garden.config as config
+from beer_garden.db.mongo.models import CommandPublishingBlockList
+
+
+def command_publishing_blocklist_add(command):
+    import beer_garden.router
+
+    if config.get("garden.name") != command["namespace"]:
+        blocked_command = CommandPublishingBlockList(**command)
+        blocked_command.save()
+        command["id"] = str(blocked_command.id)
+        beer_garden.router.route(
+            Operation(
+                operation_type="COMMAND_BLOCKLIST_ADD",
+                kwargs={"command": command},
+                target_garden_name=command["namespace"],
+            )
+        )
+        blocked_command.status = "ADD_REQUESTED"
+        blocked_command.save()
+    else:
+        blocked_command = CommandPublishingBlockList(**command)
+        blocked_command.save()
+    return blocked_command
+
+
+def command_publishing_blocklist_delete(blocked_command_id):
```

Comment:
This was okay taking an actual `blocked_command` which is a `CommandPublishingBlocklist` instance.  That helps avoid the double db lookup.